packages/wrangler/src/__tests__/r2.test.ts

```diff
@@ -148,32 +141,35 @@ describe("wrangler", () => {
 			          `);
 				});
 
-				it("should create a bucket", async () => {
-					const requests = mockCreateRequest("testBucket");
+				it("should create a bucket & check request inputs", async () => {
+					msw.use(
+						rest.post(
+							"*/accounts/:accountId/r2/buckets",
+							async (request, response, context) => {
+								const { accountId } = request.params;
+								expect(accountId).toEqual("some-account-id");
+								expect(await request.json()).toEqual({ name: "testBucket" });
```

Comment:
the `"testBucket"` assertion is related to `runWrangler("r2 bucket create testBucket")` from this test, so that would be easy to investigate a potential failure ... but where does `"some-account-id"` come from? if that assertion failed, would it be easy to investigate that assertion?

Reply:
Yes, that is part of the initial account setup for tests. It happens in one place for all tests. 
https://github.com/cloudflare/wrangler2/pull/1768/files#diff-129fe75559da917c336722e1973c0294dde22f66b3affceb26cd95ad93c4c94bL14